src/api/app/views/webui2/webui/request/_request_history.html.haml

```diff
@@ -0,0 +1,47 @@
+%ul.list-unstyled
+  %li.media
+    - user = User.find_by_login(bs_request.creator)
+    = image_tag_for(user, size: 32, custom_class: 'mr-3', width: nil)
+    .media-body
+      %h5.mt-0.mb-1 #{bs_request.creator} (#{fuzzy_time(bs_request.created_at)})
+      created request
+      .media.mt-2
+        .media-body
+          - unless bs_request.description.blank?
+            .req-description-box
+              - if bs_request.description.length > 100
+                = simple_format(bs_request.description, class: 'collapsed-history', id: 'req-description')
+                = link_to('', '###', id: "show-full-req-description", class: 'collapsed-link', title: 'show more')
+              - else
+                = simple_format(bs_request.description)
+
+  - history.each_with_index do |element, index|
+    %li.media.mt-3
+      = image_tag_for(element.user, size: 32, custom_class: 'mr-3', width: nil)
+      .media-body
+        %h5.mt-0.mb-1 #{element.user.login} (#{fuzzy_time(element.created_at)})
```

Comment:
[nitpick] repeated code.... :eyes: maybe a helper?

Reply:
> repeated code....  maybe a helper?

or partial if possible..

Reply:
partials make it *much* slower!

Reply:
i moved the media heading and body to an partial, i don't have a strong opinion on that :sweat_smile: